spec/models/user_spec.rb

```diff
@@ -155,6 +155,43 @@
     end
   end
 
+  describe 'scopes' do
+    let(:approved_affiliate_user) { users(:affiliate_manager) }
+    let(:not_approved_user) { users(:affiliate_manager_with_not_approved_status) }
+    let(:pending_user) { users(:affiliate_manager_with_pending_approval_status) }
+    let(:non_affiliate) { users(:developer) }
+    let(:inactive_approved_user) { users(:inactive_developer) }
+
+    describe '.approved_affiliate' do
+      subject(:not_approved) { User.approved_affiliate }
+
+      it { is_expected.to include(approved_affiliate_user) }
+      it { is_expected.not_to include(not_approved_user) }
+      it { is_expected.not_to include(non_affiliate) }
+      it { is_expected.not_to include(pending_user) }
+    end
+
+    describe '.not_approved' do
+      subject(:not_approved) { User.not_approved }
+
+      let(:approved_user) { users(:affiliate_manager) }
+      let(:not_approved_user) { users(:affiliate_manager_with_not_approved_status) }
+      let(:pending_user) { users(:affiliate_manager_with_pending_approval_status) }
+
+      it { is_expected.to include(not_approved_user) }
+      it { is_expected.not_to include(approved_user) }
+      it { is_expected.not_to include(pending_user) }
+    end
+
+    describe '.not_active' do
```

Comment:
@peggles2 , this is totally up to you, but I have a slight preference for `inactive` over `not_active`. I think it's more standard, and it just rolls of the...brain...a little better.  I see that the `not_active` fixture you set up is using `inactive`, as well as the spec variable: `let(:inactive_approved_user)`. We should be consistent with whichever we use.

Reply:
Ah, scratch (some of) that. I see that Authlogic uses `not_active`, so we should be consistent with that (I'm guessing why that's why you used that in the first place...). Let's just make sure we use `not_active` consistently instead of `inactive`.